File: uri/media/media.cs
using System;
using System.Globalization;

namespace SobreExercicios
{
    class MediaNum
    {
        static double peso1;
        static double peso2;


        static double calculamedia(double notaA, double notaB)
        {
            double h;
            h = ( (notaA * peso1) + (notaB * peso2) ) / 11.0;            
            return h;
        }

        public MediaNum()
        {
            peso1 = 3.5;
            peso2 = 7.5;
        }
        public void Show()
        {
            double notaA, notaB, numedia;
            notaA = Convert.ToDouble(Console.ReadLine(), CultureInfo.InvariantCulture);
            notaB = Convert.ToDouble(Console.ReadLine(), CultureInfo.InvariantCulture);
            numedia = calculamedia(notaA, notaB);
            Console.WriteLine("MEDIA = {0:0.00000}", numedia);
        }
    };    
    class Media
    {
        public static void Main(string[] args)
        {
            MediaNum foo = new MediaNum();
            foo.Show();
        }
    }    
}

File: uri/media/media.cpp

#include <iostream>
#include <iomanip>

using namespace std;

class Namedia{

    private:

        double pesoA;
        double pesoB;

        double calculamedia( double notaA, double notaB )
        {
            double poo;
            poo =  ((notaA * pesoA) + (notaB * pesoB))/ (pesoA + pesoB) ;
            return poo;
        }

    public:

        Namedia(){
            pesoA = 3.5;
            pesoB = 7.5;
        }

        void exibir()
        {
            double notaA, notaB, media;
            cin >> notaA;
            cin >> notaB;
            media = calculamedia(notaA, notaB);
            cout << "MEDIA = "<< std::fixed << setprecision(4) << media << "\n";
        }
};

int main(void)
{
    Namedia foo;
    foo.exibir();
    return 0;
}






